File: vclibrary/src/main/java/com/zxwl/vclibrary/util/ToastHelper.java
package com.zxwl.vclibrary.util;

import android.widget.Toast;

import com.huawei.opensdk.commonservice.common.LocContext;


/**
 * Created by Administrator on 2017/11/27.
 */

public class ToastHelper {

    private ToastHelper() {
        /* cannot be instantiated */
        throw new UnsupportedOperationException("cannot be instantiated");
    }

    public static boolean isShow = true;

    /**
     * 短时间显示Toast
     *
     * @param message
     */
    public static void showShort(CharSequence message) {
        if (isShow) {
            Toast mToast = Toast.makeText(LocContext.getContext(), "", Toast.LENGTH_SHORT);
            mToast.setText(message);
            mToast.show();
        }
    }

    /**
     * 短时间显示Toast
     *
     * @param message
     */
    public static void showShort(int message) {
        if (isShow) {
            Toast mToast = Toast.makeText(LocContext.getContext(), "", Toast.LENGTH_SHORT);
            mToast.setText(message);
            mToast.show();
        }
    }

    /**
     * 长时间显示Toast
     *
     * @param message
     */
    public static void showLong(CharSequence message) {
        if (isShow) {
            Toast mToast = Toast.makeText(LocContext.getContext(), "", Toast.LENGTH_SHORT);
            mToast.setText(message);
            mToast.show();
        }
    }

    /**
     * 长时间显示Toast
     *
     * @param message
     */
    public static void showLong(int message) {
        if (isShow) {
            Toast mToast = Toast.makeText(LocContext.getContext(), "", Toast.LENGTH_SHORT);
            mToast.setText(message);
            mToast.show();
        }
    }

    /**
     * 自定义显示Toast时间
     *
     * @param message
     * @param duration
     */
    public static void show(CharSequence message, int duration) {
        if (isShow) {
            Toast mToast = Toast.makeText(LocContext.getContext(), "", duration);
            mToast.setText(message);
            mToast.show();
        }
    }

    /**
     * 自定义显示Toast时间
     *
     * @param message
     * @param duration
     */
    public static void show(int message, int duration) {
        if (isShow) {
            Toast mToast = Toast.makeText(LocContext.getContext(), "", duration);
            mToast.setText(message);
            mToast.show();
        }
    }

}

File: baselibrary/src/main/java/com/hw/baselibrary/utils/ToastHelper.kt
package com.hw.baselibrary.utils

import android.widget.Toast
import com.hw.baselibrary.common.BaseApp

/**
 *author：pc-20171125
 *data:2019/11/8 11:18
 */
object ToastHelper {
    private fun ToastHelper() {
        /* cannot be instantiated */
        throw UnsupportedOperationException("cannot be instantiated")
    }

    var isShow = true

    /**
     * 短时间显示Toast
     *
     * @param message
     */
    fun showShort(message: String) {
        if (isShow)
            Toast.makeText(BaseApp.context, message, Toast.LENGTH_SHORT).show()
    }

    /**
     * 短时间显示Toast
     *
     * @param message
     */
    fun showShort(message: Int) {
        if (isShow)
            Toast.makeText(BaseApp.context, message, Toast.LENGTH_SHORT).show()
    }

    /**
     * 长时间显示Toast
     *
     * @param message
     */
    fun showLong(message: CharSequence) {
        if (isShow)
            Toast.makeText(BaseApp.context, message, Toast.LENGTH_LONG).show()
    }

    /**
     * 长时间显示Toast
     *
     * @param message
     */
    fun showLong(message: Int) {
        if (isShow)
            Toast.makeText(BaseApp.context, message, Toast.LENGTH_LONG).show()
    }

    /**
     * 自定义显示Toast时间
     *
     * @param message
     * @param duration
     */
    fun show(message: CharSequence, duration: Int) {
        if (isShow)
            Toast.makeText(BaseApp.context, message, duration).show()
    }

    /**
     * 自定义显示Toast时间
     *
     * @param message
     * @param duration
     */
    fun show(message: Int, duration: Int) {
        if (isShow)
            Toast.makeText(BaseApp.context, message, duration).show()
    }
}
